telegram/_bot.py

```diff
@@ -363,11 +363,24 @@ def __deepcopy__(self, memodict: Dict[int, object]) -> NoReturn:
         raise TypeError("Bot objects cannot be deepcopied!")
 
     def __eq__(self, other: object) -> bool:
+        """Defines equality condition for the :class:`telegram.Bot` object.
+        Two objects of this class are considered to be equal if their all parameters
+        are identical.
+
+        Returns:
+            :obj:`True` if both objects have all parameters identical. :obj:`False` otherwise.
+        """
         if isinstance(other, self.__class__):
             return self.bot == other.bot
         return False
 
     def __hash__(self) -> int:
+        """Builds a hash value for this object such that the hash of two objects is
+        equal if and only if the objects are equal in terms of :meth:`__eq__`.
+
+        Returns:
+            :obj:`int` The hash value of the object.
```

Comment:
IMO pointing to `TelegramObject.__hash__` is enough here, similar as done e.g. for https://docs.python-telegram-bot.org/en/stable/telegram.message.html#telegram.Message.de_json